bin/graftM

```diff
@@ -129,60 +132,59 @@ if __name__ == '__main__':
                                         formatter_class=CustomHelpFormatter,
                                         epilog='''            
  ##################################################################################################################################
-                                                       ~~ GraftM "graft" ~~
+                                                         ~ GraftM graft  ~ 
  ##################################################################################################################################
 
- Use GraftM "graft" to search, align and classify reads using the following pipeline:
-    
-    Detect reads with HMM/Diamond --> Align to HMM --> Place in pre-built phylogenetic tree using Maximum Likelihood
+    graft is the main analysis pipeline of GraftM. To run graft, a GraftM package (gpkg) is required, the key components of which
```

Comment:
I guess it would be good to make sure this looks well wrapped on a terminal of 80chars. I think these lines might be a bit long.